lib/Client/KeyValue.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Created by PhpStorm.
```

Comment:
Let's delete the automatic headers added by the IDE (here, LogRecord, ReportRequest)

Reply:
done